packages/apps/Settings/res/values-tr/mad_strings.xml

```diff
@@ -317,6 +324,30 @@
     <string name="menu_hide_substratum_icons">Hide icon overlays</string>
 
     <!-- Force authorize substratum packages -->
+    <string name="force_authorize_substratum_packages_title">Tüm uygulamaları temalarken doğrulamayı zorla</string>

```

Comment:
Here you start duplicating lines.
"force_authorize_substratum_packages_title" is already in the file.
Can you please remove the lines with english language?